dotnetapp-dev/README.md

```diff
@@ -26,6 +30,38 @@ docker run --rm dotnetapp-dev Hello .NET Core from Docker
 
 Note: The instructions above work for both Linux and Windows containers. The .NET Core docker images use [multi-arch tags](https://github.com/dotnet/announcements/issues/14), which abstract away different operating system choices for most use-cases.
 
+## Build and run sample for unit testing with Docker
+
+The sample runs unit tests as part of `docker build`. That's useful as a means of getting feedback during `build` (the build will fail), but there isn't an easy way to get the test logs. The sample exposes a test stage that you can build and then run explicity. Using [volume mounting](https://docs.docker.com/engine/admin/volumes/volumes/), you can cause unit tests logs to be written to your local disk for viewing outside of a container.
```

Comment:
I question whether or not the tests should be failing 'out of the box'.  I am thinking this is going to cause confusion.  Would it be better in the walk through to introduce a test failure?